cmd/krel/cmd/release_notes.go

```diff
@@ -208,7 +208,7 @@ func init() {
 	releaseNotesCmd.PersistentFlags().BoolVar(
 		&releaseNotesOpts.listReleaseNotesV2,
 		"list-v2",
-		false,
+		true,
```

Comment:
so I actually want it to be more robust -- we probably should invert the whole flag into `--list-v1` with default `false`. it's a bigger change, but I think it would be a better user experience.

if we were to accept this change and want to use v1 somehow, I think cobra would want us to do `--list-v2=false` and I would prefer seeing `--list-v1` which is more explicit / avoiding "not-something".

what do you think? 😄

Reply:
That makes sense. Will update the PR later.